pages/newanalysisform/index.js

```diff
@@ -100,81 +102,88 @@ export default function NewAnalysisForm({
   }
 
   return (
-    <form onSubmit={handleSubmit}>
-      <HeaderContainer>
-        <Button
-          type="button"
-          variant="small"
-          name="back"
-          onClick={() => history.back()}
-        >
-          ❮
-        </Button>
-        <TitleInput
-          name="title"
-          required
-          placeholder="Titel der Analyse"
-        ></TitleInput>
-        <HeaderDate>
-          <DateInput
-            type="date"
-            pattern="\d{4}-\d{2}-\d{2}"
-            defaultValue={date.toISOString().slice(0, 10)}
-            name="date"
-            max={date.toISOString().slice(0, 10)}
+    <>
+      <Head>
+        <title>Neue Analyse</title>
+        <meta charSet="UTF-8"></meta>
+      </Head>
+      <ScrollToTop />
+      <form onSubmit={handleSubmit}>
+        <HeaderContainer>
+          <Button
+            type="button"
+            variant="small"
+            name="back"
+            onClick={() => history.back()}
```

Comment:
Just as a side note, you're less likely to run into unexpected behavior when you use the `router` from next. It's based on the history API, but it's consistent with other attributes from the router object. You can leave your code as is. Just be aware that there's a solution within next too 😃

Reply:
thanks, i changed it :)